lib/urlwatch/migration.py

```diff
@@ -81,8 +65,8 @@ def migrate_cache(urlwatcher):
         print("""
     Migrating cache: {cache_dir} -> {cache_db}
     """.format(cache_dir=cache_dir, cache_db=cache))
-
         old_cache_storage = CacheDirStorage(cache_dir)
-        urlwatcher.cache_storage.restore(old_cache_storage.backup())
-        urlwatcher.cache_storage.gc([job.get_guid() for job in urlwatcher.jobs])
+        new_cache_storage = CacheMiniDBStorage(cache)
+        new_cache_storage.restore(old_cache_storage.backup())
+        new_cache_storage.close()
```

Comment:
The only functional change in this PR is here. I removed the `gc` call during migration.
This way, users using multiple url files (example use case: #148) can also safely migrate their cache.